lib/mandrill_mailer/core_mailer.rb

```diff
@@ -292,9 +292,9 @@ def apply_interceptors!(obj)
     # Makes this class act as a singleton without it actually being a singleton
     # This keeps the syntax the same as the orginal mailers so we can swap quickly if something
     # goes wrong.
-    def self.method_missing(method, *args)
+    def self.method_missing(method, *args, **kwargs)
       return super unless respond_to?(method)
-      new.method(method).call(*args)
+      new.method(method).call(*args, **kwargs)

```

Comment:
Looks like the only foolproof way to do this in all versions of Ruby (we support) is with `ruby2_keywords`, as shown in https://eregon.me/blog/2021/02/13/correct-delegation-in-ruby-2-27-3.html

Regardless we're also omitting potential block parameters which feels like something we should also fix while we're at it, right?